DESCRIPTION

```diff
@@ -17,7 +17,8 @@ RoxygenNote: 7.3.2
 Suggests: 
     testthat (>= 3.0.0),
     Matrix,
-    SeuratObject (>= 5.0.0)
+    SeuratObject (>= 5.0.0),
```

Comment:
You might also need to add a REMOTES section so that the CI knows where/how to download bpcells.

```
Remotes:
    github::bnprks/BPCells/r
```